immuni_analytics/celery/scheduled/tasks/store_operational_info.py

```diff
@@ -50,6 +53,12 @@ async def _store_operational_info() -> None:
 
     if operational_info_documents:
         OperationalInfo.objects.insert(operational_info_documents)
+        # NOTE: decrementing together to better show it has been done in the same tsaks.
```

Comment:
there is a typo in `tasks`, and I would move this comment below where you actually decrement the counter. You may also leverage `Counter` from `collections` to do something similar to the following:
```
counter = Counter(document.platform.value for document in operational_info_documents)
for platform, count in counter.items():
    OPERATIONAL_INFO_ENQUEUED.labels(platform).dec(count)
```